yoga/Yoga.cpp

```diff
@@ -1296,8 +1298,7 @@ static void YGNodeComputeFlexBasisForChild(
     if (child->getLayout().computedFlexBasis.isUndefined() ||
         (YGConfigIsExperimentalFeatureEnabled(
              child->getConfig(), YGExperimentalFeatureWebFlexBasis) &&
-         child->getLayout().computedFlexBasisGeneration !=
-             gCurrentGenerationCount)) {
+         child->getLayout().computedFlexBasisGeneration != generationCount)) {
```

Comment:
The {`computedFlexBasisGeneration`, `generationCount`} members of `YGLayout` type should be updated to 16-bit unsigned.